internal/policy/verify.go

```diff
@@ -410,6 +410,23 @@ func VerifyTag(ctx context.Context, repo *git.Repository, ids []string) map[stri
 			continue
 		}
 
+		tagRef, err := repo.Tag(strings.Trim(absPath, "refs/tags"))
```

Comment:
This could just use `repo.Reference` and avoid having to trim the prefix.

Reply:
Also, this should maybe be a part of `verifyTagEntry` where we also load the tag object.

Reply:
Also, perhaps worth double checking that the entry's ref matches tagObj.Name.

Reply:
I have moved it to verifyTagEntry, and I have added a check to double check that the entry's ref matches the tagObj.Name.

Reply:
repo.Reference still outputs in the format refs/tags/tagname, so trim would still have to be done.